### ISSUE TITLE
Searchable select not working with screenreader

### ISSUE THREAD

**Issue by thijsvdanker** [livewire/flux#2261]
### Flux version

v2.10.1

### Livewire version

v3.7.2

### Tailwind version

v4.1.18

### Browser and Operating System

Safari and Firefox on MacOS

### What is the problem?

When using the searchable select (https://fluxui.dev/components/select#searchable-select) with VoiceOver it does not announce all the options.

The combobox (https://fluxui.dev/components/select#combobox) does work as expected.

This issue looks a lot like https://github.com/livewire/flux/issues/1566 but is about the *searchable select*.

### Code snippets to replicate the problem

<h2>Listbox from searchable select example: not working with VoiceOver</h2>

```
<flux:select variant="listbox" searchable placeholder="Choose industries...">
    <flux:select.option>Photography</flux:select.option>
    <flux:select.option>Design services</flux:select.option>
    <flux:select.option>Web development</flux:select.option>
    <flux:select.option>Accounting</flux:select.option>
    <flux:select.option>Legal services</flux:select.option>
    <flux:select.option>Consulting</flux:select.option>
    <flux:select.option>Other</flux:select.option>
</flux:select>
```

<h2>Combobox from combobox example: working with VoiceOver</h2>

```
<flux:select variant="combobox" placeholder="Choose industry...">
    <flux:select.option>Photography</flux:select.option>
    <flux:select.option>Design services</flux:select.option>
    <flux:select.option>Web development</flux:select.option>
    <flux:select.option>Accounting</flux:select.option>
    <flux:select.option>Legal services</flux:select.option>
    <flux:select.option>Consulting</flux:select.option>
    <flux:select.option>Other</flux:select.option>
</flux:select>
```

### Screenshots/ screen recordings of the problem

*Sry, I'm struggling to make a screenrecording of VoiceOver*

### How do you expect it to work?

When I have focus on the [searchable select](https://fluxui.dev/components/select#searchable-select) and use arrow-down to navigate the options, VoiceOver should announce the active option.

### Please confirm (incomplete submissions will not be addressed)

- [x] I have provided easy and step-by-step instructions to reproduce the bug.
- [x] I have provided code samples as text and NOT images.
- [x] I understand my bug report will be closed if I haven't met the criteria above.

**Comment by ganyicz** [comment]
Hi @thijsvdanker,

Thanks for reporting!

I've submitted a PR with a fix.
